doc/whats_new/v0.21.rst

```diff
@@ -772,3 +772,256 @@ These changes mostly affect library developers.
   when `fit` is called twice with the same data, the ouput of
   `predict`, `predict_proba`, `transform`, and `decision_function` does not
   change. :pr:`12328` by :user:`Nicolas Hug <NicolasHug>`
+
+Code and Documentation Contributors
+-----------------------------------
+
+Thanks to everyone who has contributed to the maintenance and improvement of the
+project since version 0.20, including:
+
+1kastner, 211217613, A. Flaxman, Aaron Schumacher, Aaron Staple, Aarshay Jain,
```

Comment:
This is unbelievably long relative to previous versions. Is this because git has started recording secondary committers?

Reply:
I would say something is off there. For instance, to take a random example,  Vadim Markovtsev has 1 commit d8c363f296948a9171ac8a5d69f79dcb56589335 on Aug 24, that was also part of the 0.20.0 (and 0.20.1, 0.20.2, 0.20.3), so logically it was not added for 0.21

Reply:
Did you use the `0.20.3` tag as the reference? Actually which script generates this list? Didn't see it in `doc/developers/maintainer.rst`.

Reply:
Yes, silly error on my part. Missed the `..` in the cmd